### ISSUE TITLE
Finetuning for VCR

### ISSUE THREAD

**Issue by vateye** [thunlp/PEVL#6]
Hi, I have noticed when finetuning for VCR task, both ita and itm are used. The itm loss is computed by the cross entropy between the itm prediction and label. But for the ita loss, I think you directly regard the question and negative answer as a positive pair? Since you initialize `sim_targets = torch.zeros(sim_i2t_m.size()).to(image.device)`and
`sim_targets.fill_diagonal_(1)  `
Is this reasonable?

**Comment by vateye** [comment]
Hi, any update? 😄

**Comment by F-Yuan303** [comment]
Hi, can i ask how you get these files:
train_vcr_file: ['./2022_01_06_vcr_pretrain_prompt_data.json'] 
train_val_vcr_q2a_file: ['./pevl_vcr_itm_finetune_val_QA_data.json']
train_val_vcr_qa2r_file: ['./pevl_vcr_itm_finetune_val_QAR_data.json']
